### PR TITLE
Fix typos && broken URLs in docs, add a few missing "the"s

### LINKED ISSUE #357: Wrong source code URLs in the docs, "projet" typos
e.g. on the [Play slick page](https://www.playframework.com/documentation/2.5.x/PlaySlick)
screen shot 2016-03-07 at 11 41 12 am

The source URL is https://github.com/playframework/playframework/blob/master/documentation/manual/working/scalaGuide/main/sql/slick/PlaySlick.md, and it's a 404.

Also, there are typos: `projet` -> `project` in some places in the docs.

I can submit a PR fixing those.

### PR DESCRIPTION
**Pull request by kubukoz**
Fixes #357. Also replaced some Typesafe mentions with Lightbend.

[One URL left with a TODO](358/files#diff-6e12c50e78295138ed776e6a5e042d46R136), because I don't know where it should point to.

### CODE REVIEW

File: docs/manual/working/scalaGuide/main/sql/slick/PlaySlick.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Injecting a `DatabaseConfig` instance for a different database is also easy. Sim
 
 Of course, you should replace the string `"<db-name>"` with the name of the database's configuration you want to use.
 
-For a full example, have a look at [this sample projet](https://github.com/playframework/play-slick/tree/master/samples/di).
+For a full example, have a look at [this sample project](https://github.com/playframework/play-slick/tree/master/samples/di). TODO 404
```
Comment (marcospereira) [inline docs/manual/working/scalaGuide/main/sql/slick/PlaySlick.md]: You can point to https://github.com/playframework/play-slick/tree/2.0.0/samples/basic. This version uses Dependency Injection.

Comment (kubukoz) [inline docs/manual/working/scalaGuide/main/sql/slick/PlaySlick.md]: Done!

**Comment by marcospereira** [comment]
Thanks, @kubukoz. Could you please squash your commits into a single one? After that, this PR will be ready to merge. ;-)

**Comment by kubukoz** [comment]
Done!

**Pull request merged**